website/docs/setup/configuration.md

```diff
@@ -127,6 +132,24 @@ directory within the container.
 </TabItem>
 </Tabs>
 
+Similar to environment variables, user can specify the values in config file.
```

Comment:
This section is unclear. How should this file be used? It's also outside of a Tab group and therefore is very platform specific. Please remove or add more docs and instructions for every platform.

Reply:
Information about how to use the config file is already mentioned. And I have updated the docs in each platform respectively. Please do let me know if any other changes are required.